packages/yourloops/components/header-bars/banner.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2023, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React, { type FC } from 'react'
+
+import { useTheme } from '@mui/material/styles'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import { ConfigService } from '../../lib/config/config.service'
+
+
+export const Banner: FC = () => {
+  const theme = useTheme();
+
+  return (
+    <Box
+      alignItems="center"
+      bgcolor="var(--primary-color-dark)"
+      display="flex"
+      justifyContent="center"
+      minHeight={60}
+      paddingLeft={theme.spacing(4)}
+      paddingRight={theme.spacing(4)}
+      width="100%"
+    >
+      <Typography
+        color={theme.palette.common.white}
+        fontSize={14}
```

Comment:
I started looking for the variant and I realized that I was spending way too much time finding the right one instead of having one more line of code... I still don't see the point of using this when it is already simple and easier to read (when reading `variant={"subtitle2"}` I don't really know what to expect, but when reading `fontSize={14} fontWeight="medium"` it is pretty obvious).